src/python/tensorflow_cloud/core/run.py

```diff
@@ -117,6 +118,7 @@ def run(
             up to 64 key-value pairs in lowercase letters and numbers, where
             the first character must be lowercase letter. For more details see
             https://cloud.google.com/ai-platform/training/docs/resource-labels.
+        job_dir: Optional. A Google Cloud Storage path in which to store training outputs.
```

Comment:
Can you add a line explaining why this is needed. Why not skip this param altogether and save and load model like shown in this example: https://github.com/tensorflow/cloud/blob/master/src/python/tensorflow_cloud/core/tests/examples/call_run_within_script_with_keras_fit.py

Reply:
I thought adding the `jobDir` would enable other feature in Cloud AI Platform but it is not.

It only enables the "Download Model" button which copy a gsutil command to run to download the content in jobDir.

I will drop this PR.